### PR TITLE
latexindent 3.13.5

### PR DESCRIPTION
**Pull request by branchv**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/latexindent.rb [hunk 1]
```diff
@@ -84,8 +84,8 @@ class Latexindent < Formula
   end
 
   resource "MRO::Compat" do
-    url "https://cpan.metacpan.org/authors/id/H/HA/HAARG/MRO-Compat-0.14_01.tar.gz"
-    sha256 "bc214d7964bc72f5a4015cc6b0d27376071cb64bd955280fea40c046b64d911a"
+    url "https://cpan.metacpan.org/authors/id/H/HA/HAARG/MRO-Compat-0.13.tar.gz"
```
Comment (branchvincent) [inline Formula/latexindent.rb:87]: seems `0.13` is the latest stable version: https://metacpan.org/pod/MRO::Compat

Comment (chenrui333) [reply Formula/latexindent.rb:87]: interesting, could that be the `0.14_01` not properly released?

Comment (chenrui333) [reply Formula/latexindent.rb:87]: tag is in the github repo.

Comment (chenrui333) [reply Formula/latexindent.rb:87]: cc @haarg (do you mind weighing in here?)

Comment (haarg) [reply Formula/latexindent.rb:87]: The convention on CPAN is that versions with underscores indicate beta releases, so they aren't indexed.

I've just released 0.15, a new stable release.

Comment (branchvincent) [reply Formula/latexindent.rb]: ah awesome, thanks for your help!

Comment (chenrui333) [reply Formula/latexindent.rb]: Thanks haarg!

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**